### PR TITLE
APPOPS-17333 Vue moved to tier 1

### PR DESCRIPTION
**Pull request by jakub-jarek**
- (unused) vue3 docs moved to (used) vue file, updated based on xdn functionality
- next commerce removed from tier 1 menu, vue added
- extracted duplicate text into vars, used in next and vue -> may be used elsewhere also, but should ideally be done as part of another task / when other frameworks are touched

### CODE REVIEW

File: constants.js [hunk 1]
```diff
@@ -94,6 +94,29 @@ yarn global add ${PACKAGE_NAME}/cli
...
+const INIT_DEFAULT_PACKAGES = `
+- The \`${PACKAGE_NAME}/cli\` package - Allows you to control ${PRODUCT} through project-local CLI.
+- The \`${PACKAGE_NAME}/core\` package - Allows you to declare routes and deploy your application to ${PRODUCT}.
+- The \`${PACKAGE_NAME}/prefetch\` package - Allows you to configure a service worker to prefetch and cache pages to improve browsing speed.
+- The \`${PACKAGE_NAME}/devtools\` package - Allows you to monitor the caching and prefetching functionality.
+`;
+
+const INIT_TIER1_FILES = `
+- \`${CONFIG_FILE}\` - Contains various configuration options for ${PRODUCT}.
+- \`routes.js\` - A default routes file that proxies all requests to the server. Update this file to add caching or proxy some URLs to a different origin.
```
Comment (tristanlee85) [inline constants.js:106]: @jakub-jarek perhaps this variable is named incorrectly. Wouldn't this be better described as `INIT_DEFAULT_FILES`?

Comment (jakub-jarek) [reply constants.js:106]: done

File: src/pages/guides/sites_frameworks/getting_started/vue.md [hunk 2]
```diff
@@ -2,205 +2,141 @@
...
+  siteUrl="https://layer0-docs-layer0-vue3-example-default.layer0-limelight.link/"
   deployFromRepo />
 
-{{ PREREQ }}
-
-## Create a new Vue.js app {/*create-a-new-vuejs-app*/}
+## Connector {/*connector*/}
 
-If you don't already have a Vue.js app, create one by using the Vue CLI:
+This framework has a connector developed for {{ PRODUCT }}. See [Connectors](connectors) for more information.
```
Comment (jsernaque) [inline src/pages/guides/sites_frameworks/getting_started/vue.md:20]:
```suggestion
{{ PRODUCT }} provides a connector for this framework. [Learn more.](/guides/sites_frameworks/connectors)
```

**Comment by petrvecera** [comment]
@jsernaque , @tristanlee85  can we get this released ? Thanks 🙏

**Review by tristanlee85 — DISMISSED** [review]

File: src/pages/guides/sites_frameworks/getting_started/vue.md [hunk 3]
```diff
@@ -2,196 +2,136 @@
...
   deployFromRepo />
 
-{{ PREREQ }}
+## Connector {/*connector*/}
 
-## Create a new Vue.js app {/*create-a-new-vuejs-app*/}
+{{ PRODUCT }} provides a connector for this framework. [Learn more.](/guides/sites_frameworks/connectors)
 
-If you don't already have a Vue.js app, create one by using the Vue CLI:
+<ButtonLink variant="stroke" type="code" withIcon={true} href="https://github.com/edgio-docs/edgio-connectors/tree/main/edgio-vue-connector">
```
Comment (jsernaque) [inline src/pages/guides/sites_frameworks/getting_started/vue.md:22]: This URL returns a 404. I found the following 3 vue connectors:
https://github.com/edgio-docs/edgio-connectors/tree/main/edgio-vue-3-connector
https://github.com/edgio-docs/edgio-connectors/tree/main/edgio-vue-cva-connector
https://github.com/edgio-docs/edgio-connectors/tree/main/edgio-vue-storefront-connector

The vue 3 connector indicates that we should use vue-cva. If that is correct, then we should update this URL to point to the vue-cva URL.

**Review by jsernaque — APPROVED** [review]

**Pull request merged**